README.md

```diff
@@ -2,22 +2,24 @@
 
 # Vello
 
-**An experimental GPU compute-centric 2D renderer**
+**An experimental GPU compute-centric 2D renderer.**
```

Comment:
For awareness, I use a fullstop here because it causes a markdown lint, `MD036/no-emphasis-as-heading: Emphasis used instead of a heading`.

<img width="1287" alt="image" src="https://github.com/linebender/vello/assets/48108917/b2b05a46-6dcb-4014-9df3-308ff0b1523a">

Nonetheless I removed it at your request.